plyflatten/utils.py

```diff
@@ -36,12 +50,18 @@ def utm_zone_from_ply(ply_path):
         if s:
```

Comment:
this block of code is now duplicated (i.e. in `epsg_code_from_comments` and in `crs_from_ply`). Could we deduplicate it by making `epsg_code_from_comments` more generic?